text_localization_environment/__init__.py

```diff
@@ -0,0 +1,7 @@
+from gym.envs.registration import register
+from text_localization_environment.TextLocEnv import TextLocEnv
+
+register(
+    id='TextLocEnv-v0',
+    entry_point='text_localization_environment.environment:TextLocEnv',
+)
```

Comment:
Please add a trailing newline to the file. 🙂

Reply:
Fixed